### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by F3Joule**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+-- \
+--execution=wasm \
+--chain=kusama
+```
+
+## Using docker
+
+Official docker hub image of Subsocial parachain: https://hub.docker.com/r/dappforce/subsocial-parachain
+
+Simple run and join the network with docker:
```
Comment (Yung-Beef) [inline README.md:45]: Simply

File: README.md [hunk 2]
```diff
@@ -1 +1,53 @@
...
+--name=your-node-name \
+--pruning=archive \
+-- \
+--execution=wasm \
+--chain=kusama
+```
+
+## Using docker
+
+Official docker hub image of Subsocial parachain: https://hub.docker.com/r/dappforce/subsocial-parachain
```
Comment (Yung-Beef) [inline README.md:43]: the Subsocial parachain

File: README.md [hunk 3]
```diff
@@ -1 +1,53 @@
-# Subsocial Parachain
+# Subsocial parachain node
+
+The Subsocial parachain is our custom built Kusama blockchain, based on the Substrate framework.
+
+## Build
+
+For linux, FreeBSD, OpenBSD and macOS:
```
Comment (siman) [inline README.md:7]:
```suggestion
For Linux, FreeBSD, OpenBSD, and macOS:
```

File: README.md [hunk 4]
```diff
@@ -1 +1,53 @@
...
+--name=your-node-name \
+--pruning=archive \
+-- \
+--execution=wasm \
+--chain=kusama
+```
+
+## Using docker
+
+Official docker hub image of the Subsocial parachain: https://hub.docker.com/r/dappforce/subsocial-parachain
```
Comment (siman) [inline README.md:43]:
```suggestion
Official Docker Hub image of the Subsocial parachain: https://hub.docker.com/r/dappforce/subsocial-parachain
```

**Review by siman — CHANGES_REQUESTED** [review]

**Review by siman — APPROVED** [review]

**Pull request merged**